keyboards/handwired/unk/rev1/config.h

```diff
@@ -30,6 +44,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* COL2ROW or ROW2COL */
 #define DIODE_DIRECTION COL2ROW
 
+/*
+ * Split Keyboard specific options, make sure you have 'SPLIT_KEYBOARD = yes' in your rules.mk, and define SOFT_SERIAL_PIN.
+ */
+
+#define USE_SERIAL
+
+// Because we are using OLED on I2C.
+// USE_I2C does not to be defined here since OLED_DRIVER_ENABLE has been set in rules.mk
+#define SOFT_SERIAL_PIN D3
+#define SELECT_SOFT_SERIAL_SPEED 1
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+// #define BACKLIGHT_LEVELS 3
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
```

Comment:
Why are you adding this back? This is all present in `info.json`, and does not belong in `config.h` anymore.